app/javascript/packs/load_more.js

```diff
@@ -0,0 +1,72 @@
+
+// console.log(13 % 3) // 1 залишок від ділення
+// console.log(parseInt((13/3))); // 4 цілочисельне ділення
+// console.log(Math.ceil(13/3));  // 5 округлення до більшого
+
+// let loadMoreBtn = document.querySelector('.button-load-more');
+var loadMoreBtnNews = document.querySelector('.btn-news');
+var newsContainer = document.querySelector('.news-container');
+var currentItemsNews = 6;
+var loadMoreBtnDocs = document.querySelector('.btn-docs');
+var docsContainer = document.querySelector('.docs-container');
+var currentItemsDocs = 9;
```

Comment:
please, use let instead of var

Reply:
An error occurs when using let instead of var. This script is from another PR. Needs refactoring.

Reply:
fixed in another PR
https://github.com/mmrshk/innovation_forpost/blob/488d3507c2f5d314f113cf4640658e00ffa54e20/app/javascript/packs/load_more.js#L5-L9